tests/dogtag/pytest-ansible/provision/post_provision.yml

```diff
@@ -0,0 +1,21 @@
+- hosts: master
+  gather_facts: true
+
+  tasks:
+
+    - name: Install list of packages for CS Master for Fedora
+      dnf : pkg={{item}} state=latest
+      with_items:
+        - setools
+        - dnf-plugins-core
+      when: ansible_distribution == "Fedora"
+
+# The installation is breaking with pki/master with ssl handshake error.
+# Team is working on the fix. Would uncomment below as soon as it gets fixed.
+#    - name: set PKI master copr repo
+#      shell: dnf copr enable @pki/master -y
+#      when: ansible_distribution == "Fedora"
+
+    - name: disable selinux
+      selinux:
+        state: disabled
```

Comment:
Minor: Why are we turning off selinux? Do we have any known bugs that cause failure in PKI?

Reply:
Fixed